DS-3672: Add 'out of bag' for R-squared [revdep skip]

chrisfacer opened the pull request:
Add 'out of bag' for R-squared [revdep skip]

Commit:
Add 'out of bag' for R-squared [revdep skip]

Add 'out of bag' for R-squared [revdep skip]

jrwishart reviewed (changes requested):
Looks good. Only minor issue is the unit tests look like they need an update. Seems the deep learning feature returns slightly different labels now (probably some `keras` update) and the unit test needs updating for the travis build. Also worth forcing the survey version in the travis file too to avoid the latest survey causing an error.

Commit:
Try to resolve Travis issues

Try to resolve Travis issues
- Update handling of chartData attribute in tests for Deep Learning.
- Ensure survey package installed at v 4.0
